README.md

```diff
@@ -1,2 +1,139 @@
 # github-security-jira
-Github Action for integrating Security Alerts with JIRA
+GitHub Action for mapping security alerts to Jira tickets.
+
+
+## Setup
+
+This action is designed to run in conjunction with another action. In summary, you need the following pieces set up to sync alerts with Jira:
+
+1. Two repo secrets containing a GitHub access token and a Jira API token, respectively.
+2. A YAML file which configures the settings for the Jira project where tickets should be created.
+3. A workflow file which runs the two actions on a schedule, continually creating new tickets when necessary.
+
+
+### Jira API Token
+The `reload/github-security-jira` action requires a secret called `JiraApiToken` which should contain an API Token for the Jira user that should be used to create tickets. The token is passed to the action in the `JIRA_TOKEN` environment variable.
+
+
+### Jira config file `gh-jira.yaml`
```

Comment:
Maybe it's just me, but I'd prefer if this was configured on the action in the workflow. The minimal configuration is host, user and project, and it seems more suitable in the workflow.

Reply:
Yes, that's probably a better fit actually.